tests/test_utils.py

```diff
@@ -3,7 +3,7 @@
 from schematools.utils import get_dataset_prefix_from_path, to_snake_case, toCamelCase
 
 
-def test_toCamelCase():
+def test_toCamelCase() -> None:
```

Comment:
🤯  @gkoller  kunnen we dit uitzetten? helemaal voor de tests...

Reply:
Yes and no. ;-)

Yes, to turn it off, add the following to `mypy.ini` or to `setup.cfg` after @ngaranko has migrated the mypy settings to `setup.cfg`:

```ini
[mypy-tests.*]
disallow_untyped_defs = False
disallow_incomplete_defs = False
disallow_untyped_decorators = False
```

No, as there is a school of thought, that I happen to adhere to, that states that your test code should follow the same quality standards as regular code. In case of Python code that would mean type annotated and documented (docstrings) test functions. After all, why should the test code be any different than regular code? Is it not as important? Don't you want to know the type of fixtures, so that you know what methods and attribute references can be made on them. Don't you want to know what exactly is being tested? I for sure want to know what kind of edge case or regression is being tested. So that at some point in time I can easily determine that the test is no longer required, or should definitely be kept.

I find that being as strict with test functions as with regular code leads to more meaningful tests and hence better test suites. The extra work required for a test to be written, means it will only be written if really serves a purpose. It cuts down on useless or no longer relevant tests.

Not saying that is the case here, but I'd rather see us getting into the habit. This is still a young project. Once we're a couple of years in, the extra effort will have paid of.

Reply:
For what it is worth I haven't even enabled all the flake8 warnings that I think should be turned on (eg docstrings for test functions)

Reply:
Even if you want to have the same type strictness, mypy really won't like your pytest fixtures. will you be annotating each one of them? This increases test complexity.

(FYI, pycharm can resolve the fixture types if you set the test framework to "pytest" in your project)